js/tests/redux/reducers/warning.test.js

```diff
@@ -0,0 +1,22 @@
+/* global describe, it, expect */
+
+import { setWarningMessage } from "../../../src/redux/actions/warning";
+import warning from "../../../src/redux/reducers/warning";
+
+describe( "cornerstone reducers", () => {
```

Comment:
I think all the descriptions referring to `cornerstone` need to be changed to the correct description.